src/icrar/leap-accelerate/model/cpu/MetaData.cc

```diff
@@ -130,7 +130,7 @@ namespace cpu
         casacore::Vector<double> time = msmc->time().getColumn();
         double epoch = time[0];
         int nEpochs = 0;
-        for(int i = 0; i < time.size(); i++)
+        for(size_t i = 0; i < time.size(); i++)
```

Comment:
great to see all these `int` -> `size_t` replacements, they remove quite a few warnings. Eventually we could get warning-free, then turn them all on and as errors by default.

Reply:
That is the plan, the only warnings really stumping me now is a second set of cuda warnings generated by gcc, but I might solve that one last then add a compiler warning as errors settings file if necessary.